_pages/algorithm-development/languages/python.md

```diff
@@ -126,49 +141,36 @@ def apply(input):
         # Raise helpful error message
         raise AlgorithmError(
             "Please provide a valid input that includes an array of numbers in the field 'numbers'")
-
 {% endhighlight %}
 
-Go ahead and type or paste the code sample above in the Algorithmia code editor after removing the "Hello World" code.
-
-Now compile the new code sample and when that's done test the code in the console by passing in the input and hitting enter on your keyboard:
+If we were to run this code as part of our algorithm, we should see the minimum and maximum of the numbers in the list returned in the console:
 
-{% highlight python %}
-{"numbers": [1, 4, 2, 6, 3]}
-{% endhighlight %}
-
-You should see the minimum and maximum of the numbers in the list returned in the console:
 {% highlight python %}
 {"max_num":6, "min_num":1}
 {% endhighlight %}
 
 ### Working with Data Stored on Algorithmia
 
-This next code snippet shows how to create an algorithm working with a data file that a user has stored using Algorithmia's [Hosted Data Source]({{site.baseurl}}/data/hosted).
+This next code snippet shows how to create an algorithm that works with a data file stored using Algorithmia's [Hosted Data Source]({{site.baseurl}}/data/hosted).
```

Comment:
suggest changing wording to:
This next code snippet shows how to create an algorithm that works with a data file stored in a [Hosted Data Collection]({{site.baseurl}}/data/hosted) on Algorithmia.